dockers/docker-ptf-saithrift/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM docker-ptf
+
+RUN apt-get update
+
+COPY ["deps/saithrift-0.9.tar.gz", "/deps/"]
+COPY ["*.ini", "/etc/ptf/"]
+COPY ["deps/SAI/test/saithrift/tests/*", "/usr/share/ptf-tests/"]
+
+RUN tar xzf /deps/saithrift-0.9.tar.gz -C /usr/share/; \
+    cd usr/share/saithrift-0.9;   \
+    python setup.py install;      \
+    rm -rf /deps
```

Comment:
suggest to put all these into docker-ptf since there are the same for different platforms.